server/src/database/boards.go

```diff
@@ -172,3 +173,17 @@ func (d *Database) GetBoard(id uuid.UUID) (Board, error) {
 	err := d.db.NewSelect().Model(&board).Where("id = ?", id).Scan(context.Background())
 	return board, err
 }
+
+func (d *Database) GetUserBoards(userID uuid.UUID) ([]Board, error) {
```

Comment:
Imo, the name should only be `GetBoards`, because with the user id as a parameter it is clear that these are the boards of a user